qiskit/providers/aer/backends/aer_simulator.py

```diff
@@ -337,23 +337,27 @@ class AerSimulator(AerBackend):
             'rzz', 'rzx', 'ccx', 'cswap', 'mcx', 'mcy', 'mcz', 'mcsx',
             'mcp', 'mcphase', 'mcu', 'mcu1', 'mcu2', 'mcu3', 'mcrx', 'mcry', 'mcrz',
             'mcr', 'mcswap', 'unitary', 'diagonal', 'multiplexer',
-            'initialize', 'delay', 'pauli', 'mcx_gray'
+            'initialize', 'delay', 'pauli', 'mcx_gray',
+            'jump', 'mark'
```

Comment:
If the compiler becomes part of the internal assemble we dont need to add these custom internal gates to the basis gates. These are only needed for terra transpiler, which should only be applied to circuits that don't contain these instructions.